includes/Plugin.php

```diff
@@ -145,26 +145,6 @@ protected function get_service_classes(): array {
 		return self::SERVICES;
 	}
 
-	/**
-	 * Get the bindings for the dependency injector.
-	 *
-	 * The bindings array contains a map of <interface> => <implementation>
-	 * mappings, both of which should be fully qualified class names (FQCNs).
-	 *
-	 * The <interface> does not need to be the actual PHP `interface` language
-	 * construct, it can be a `class` as well.
-	 *
-	 * Whenever you ask the injector to "make()" an <interface>, it will resolve
-	 * these mappings and return an instance of the final <class> it found.
-	 *
-	 * @since 1.6.0
-	 *
-	 * @return array<string> Associative array of fully qualified class names.
-	 */
-	protected function get_bindings(): array {
```

Comment:
Why remove this?

Reply:
Because it's identical to the parent `\Google\Web_Stories\Infrastructure\ServiceBasedPlugin::get_bindings()` method and thus can be safely removed.